src/css/base.scss

```diff
@@ -2485,3 +2507,7 @@ textarea::placeholder {
         }
     }
 }
+
+.error-input {
+    border: 1px solid var(--R500) !important;
```

Comment:
Instead of this can we try class combination -> `dc__border er-5`?

Reply:
Done